### PR TITLE
Mostly spelling fixes

### PR DESCRIPTION
**Pull request by jtu0**

### CODE REVIEW

File: content/english/about/_index.md [hunk 1]
```diff
@@ -2,32 +2,29 @@
 title: About Us
 bg_image: images/OaklandPanorama2.jpg
 image: images/oakland_skyline_lake_merritt.jpg
-description: A group of Oakland citizens who are trying to figure out what the hell
-  is going on.
+description: A group of Oakland residents who want to share what they're learning about how our city works
```
Comment (jtu0) [inline content/english/about/_index.md:5]: Moved language from `citizens` to `residents` to try to make it more inclusive.

File: content/english/about/_index.md [hunk 2]
```diff
@@ -2,32 +2,29 @@
...
 
 We want to do more.
 
-This site was made for us to be more effective at "last mile" civic engagement and enable more folks to join us. There's a LOT to take in, so we made this website to make it easier to keep track of what's going on. We will build this site out over time, particularly after the election, as we keep track of what promises were made by which politicians.
+This site was made for us to be more effective at "last mile" civic engagement and enable more folks to join us. There's a LOT to take in, so we made this website to make it easier to keep track of what's going on. We will build this site out over time, particularly after the election, as we track promises made by electeds and appointees.
 
-If you'd like to join us, please shoot us an email at XXX@XXX.com.
+If you'd like to join us, please shoot us an email at info@oaktmg.club.
```
Comment (jtu0) [inline content/english/about/_index.md]: We can also use `oakmtg@gmail.com` if you prefer

File: data/en/homepage.yml [hunk 3]
```diff
@@ -5,17 +5,17 @@ slider:
...
       button:
         enable : true
         label : "Learn more"
         link : "about"
-       
-       
-    - title : "The Last-Minute Voter's Guide to the Oakland Election"
+
+
+    - title : "The Informed Voter's Guide to the 2020 Oakland Election"
```
Comment (jtu0) [inline data/en/homepage.yml:18]: Probably should've put this in https://github.com/astropuffin/oakmtg.club/pull/26 but I didn't find it until I was doing a spelling pass

**Comment by jtu0** [comment]
@astropuffin wish I could just tag you in as a reviewer instead of in a comment!

File: content/english/about/_index.md (hunk 2 above)
Comment (astropuffin) [reply content/english/about/_index.md]: Should update to the new Google group.

**Review by astropuffin — APPROVED** [review]

**Pull request merged**